example/e2e/detoxTest.spec.js

```diff
@@ -186,6 +186,58 @@ describe('Example', () => {
       }
       await expect(getTimeText()).toHaveText('09:30');
     });
+
+    it('should let you pick tomorrow but not yesterday when setting min/max', async () => {
+      await element(by.id('DateTimePickerScrollView')).scrollTo('bottom');
+      await element(by.id('setTzOffsetToZero')).tap();
+      await elementById('setMinMax').tap();
+
+      if (isIOS()) {
+        const testElement = getDateTimePickerControlIOS();
+
+        // Ensure you can't select yesterday (iOS)
+        await testElement.setDatePickerDate('2021-11-12', 'yyyy-MM-dd');
+        await expect(getDateText()).toHaveText('11/13/2021');
+
+        // Ensure you can select tomorrow (iOS)
+        await userOpensPicker({mode: 'date', display: getPickerDisplay()});
+        await testElement.setDatePickerDate('2021-11-14', 'yyyy-MM-dd');
+      } else {
+        const uiDevice = device.getUiDevice();
+
+        // Ensure you can't select yesterday (Android)
+        const focusTwelethOfNovemberInCalendar = async () => {
+          await uiDevice.pressDPadDown();
```

Comment:
this seems like a simple for loop from 0 to n would already make sense here 🙂 
same thing below